addons/tutorial_04_split_tunneling3/resetAppPermissions.js

```diff
@@ -0,0 +1,14 @@
+(function(api, condition) {
+// Let's reset the disabled app list to have the first one item in the list
+// always disabled.  Because we are doing at the beginning of the tutorial, the
+// settings-rollback will take care of restoring the previous config.
+api.connectSignal(api.addon, 'playingChanged', () => {
+  if (api.addon.playing) {
+    api.settings.vpnDisabledApps = [];
+    api.settings.protectSelectedApps = false;
+  }
+});
+
+condition.enable();
+});
+
```

Comment:
This doesn't seem to be working. After selecting the first checkbox, and running the tutorial, the checkbox is still selected in the tutorial.

Also, do we still need the `protectSelectedApps` part if we no longer have the toggle?

Reply:
updated